protocol-designer/src/flex-components/protocol-editor/FlexModules.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react'
+import { Flex, DIRECTION_COLUMN, SPACING } from '@opentrons/components'
+import { i18n } from '../../localization'
+import { StyledText } from './StyledText'
+import styles from './FlexComponents.css'
+
+import { ModuleType, ModuleModel } from '@opentrons/shared-data'
+import { ModuleOnDeck } from 'protocol-designer/src/step-forms/types'
+import { ModelModuleInfo } from 'protocol-designer/src/components/EditModules'
+import { ModuleDiagram } from 'protocol-designer/src/components/modules'
```

Comment:
lets clean up these imports a bit:

```suggestion
import { ModelModuleInfo } from '../../components/EditModules'
import { ModuleDiagram } from '../../components/modules'
import type { ModuleOnDeck } from '../../step-forms/types'
```